app/assets/javascripts/modules/Contacts.js

```diff
@@ -0,0 +1,62 @@
+moj.Modules.Contacts = {
+    $button : $("#open-button"),
+    $remote_content : $("#remote-content"),
+    $dialog : $("#dialog-content").dialog({
+        autoOpen: false,
+        height: 400,
+        width: 600,
```

Comment:
One thing I was going to do on implementation is adjust this for responsive. Just letting you know at this point because the remediation PR doesn't include this so don't be surprised. I think we can do this as another iteration though.

Reply:
Okay cool I'll leave this for now.